<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>198</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Sub Contractor</string>
  </property>
  <widget class="QPushButton" name="settingSave_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Settings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubRoad_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub  House:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubName_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubEmail_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubCounty_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubArea_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubPostcode_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="settingSubHouse_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>County :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub Email :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Road Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Area  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Postcode :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
